pgcli/pgtoolbar.py

```diff
@@ -10,6 +10,7 @@
     InputMode.INSERT: "I",
     InputMode.NAVIGATION: "N",
     InputMode.REPLACE: "R",
+    InputMode.REPLACE_SINGLE: "R",
```

Comment:
Prompt toolkit <= 3.0.5 doesn't have InputMode.REPLACE_SINGLE, and higher versions should be fixed by the code below.